### PR TITLE
Add api key to algolia search

### PR DESCRIPTION
**Pull request by KanishkKhurana**
as per the discussion , the env variables have been edited.

### CODE REVIEW

**Comment by sansyrox** [comment]
@KanishkKhurana 👋 

Can you please rename this to `ALGOLIA_INDEX_NAME` ?                     
```
indexName: process.env.NEXT_PUBLIC_DOCSEARCH_INDEX_NAME,
```
https://github.com/sparckles/docs-base/blob/4cd67baacc500874e875b438bbc980c495733234/src/components/documentation/Search.jsx#L56

File: src/components/Testimonials.jsx
Comment (sansyrox) [inline src/components/Testimonials.jsx:1]: Love these!

File: src/components/documentation/Search.jsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import algoliasearch from 'algoliasearch/lite'
 import clsx from 'clsx'
```
Comment (sansyrox) [inline src/components/documentation/Search.jsx:7]: It is still not working. I am not sure why 😅

File: src/pages/index.jsx [hunk 2]
```diff
@@ -434,45 +436,51 @@ export default function Home({ articles }) {
...
               <div className="text-center">
                 <h2 className="text-3xl font-bold tracking-tight text-white sm:text-4xl">
-                  Some of our stats
+                  Sponsors!
                 </h2>
                 <p className="mt-4 text-lg leading-8 text-gray-300">
-                  Lorem ipsum dolor sit amet consect adipisicing possimus.
+                  Robyn is powered by these amazing people
                 </p>
               </div>
```
Comment (sansyrox) [inline src/pages/index.jsx:444]: Let's keep the stats here and move the sponsors to the `/community` page please?

**Comment by sansyrox** [comment]
Screen Shot 2023-08-08 at 23 07 12

Also, two things - the background color doesn't match the background of the website. Can we fix that?
And second, all the images are not proportional. Can we make them proportionate?

**Comment by sansyrox** [comment]
Hey @KanishkKhurana 👋 

Great work on the PR 😄 I just have a few suggestions. Please do let me know if require any further information from me 😄

**Comment by KanishkKhurana** [comment]
> Screen Shot 2023-08-08 at 23 07 12
> Also, two things - the background color doesn't match the background of the website. Can we fix that? And second, all the images are not proportional. Can we make them proportionate?

the thing is that all these photos are of different shapes. if we keep them all as circles that would look good on the theme and they all will be proportional as well. for this we would need to get a new digitalocean link. im happy to search that logo and share for approval. please let me know if this strategy works for you

**Comment by sansyrox** [comment]
Hey @KanishkKhurana :wave: 

Great work! Especially with the sponsors section. I can share the allowed logo of digital ocean with you :D

And one thing

The colors are still not fixed here - 
image

**Comment by sansyrox** [comment]
@KanishkKhurana :wave: 

You can use this link - https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/SVG/DO_Logo_vertical_blue.svg

**Comment by sansyrox** [comment]
Alsp, can we use the `#666666` color for grey here ?

image

**Comment by KanishkKhurana** [comment]
Thanks for sharing the logo of DigitalOcean. I will fix all the colors asap!

**Comment by KanishkKhurana** [comment]
> @KanishkKhurana 👋
> 
> You can use this link - https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/SVG/DO_Logo_vertical_blue.svg



Done!
Screenshot 2023-08-13 at 5 22 47 AM

**Comment by KanishkKhurana** [comment]
> Alsp, can we use the `#666666` color for grey here ?
> 
> image



Done!
Screenshot 2023-08-13 at 5 27 05 AM

**Comment by KanishkKhurana** [comment]
> Hey @KanishkKhurana 👋
> 
> Great work! Especially with the sponsors section. I can share the allowed logo of digital ocean with you :D
> 
> And one thing
> 
> The colors are still not fixed here - image



Done!
Screenshot 2023-08-13 at 5 52 38 AM

File: src/pages/index.jsx [hunk 3]
```diff
@@ -434,43 +436,62 @@ export default function Home({ articles }) {
...
-                <div className="flex flex-col bg-white/5 p-8">
+              <dl className="mt-16 grid grid-cols-1  gap-0.5 overflow-hidden rounded-2xl text-center sm:grid-cols-2 lg:grid-cols-4">
+                <div className="flex flex-col justify-center bg-white/5 p-8">
                   <dt className="text-sm font-semibold leading-6 text-gray-300">
-                    PyPi installs
+                    Robyn installs
                   </dt>
+                  {/* <a href="https://www.digitalocean.com/?refcode=3f2b9fd4968d&utm_campaign=Referral_Invite&utm_medium=Referral_Program&utm_source=badge" target="_blank" rel="noopener noreferrer">
                   <dd className="order-first text-3xl font-semibold tracking-tight text-white">
-                    850,000+
```
Comment (sansyrox) [inline src/pages/index.jsx:449]: @KanishkKhurana , can we please align the original numbers?

File: src/components/releases/mdx.jsx [hunk 4]
```diff
@@ -43,7 +43,7 @@ function ContentWrapper({ className, children }) {
       <div className="ml-10 lg:flex lg:w-full lg:justify-center ">
         <div
           className={clsx(
-            'mx-auto max-w-lg lg:mx-0 lg:w-0 lg:max-w-xl lg:flex-auto',
+            'mx-auto max-w-lg lg:mx-0 lg:w-0 lg:max-w-xl lg:flex-auto text-[#666666]   ',
```
Comment (sansyrox) [inline src/components/releases/mdx.jsx]: one last suggestion here 

```suggestion
            'mx-auto max-w-lg lg:mx-0 lg:w-0 lg:max-w-xl lg:flex-auto text-gray-300 ',
```

**Comment by sansyrox** [comment]
@KanishkKhurana 👋 

The colors are still not fixed for the light mode 😓 


Screen Shot 2023-08-14 at 16 26 38



I am talking about the font colors here.

**Comment by KanishkKhurana** [comment]
> @KanishkKhurana 👋
> 
> The colors are still not fixed for the light mode 😓
> 
> Screen Shot 2023-08-14 at 16 26 38
> I am talking about the font colors here.

fixing right away!

**Comment by KanishkKhurana** [comment]
> > @KanishkKhurana 👋
> > The colors are still not fixed for the light mode 😓
> > Screen Shot 2023-08-14 at 16 26 38
> > I am talking about the font colors here.
> 
> fixing right away!


done!

Screenshot 2023-08-14 at 2 38 20 PM

**Comment by sansyrox** [comment]
Hey @KanishkKhurana 👋 

Great work mate. There are just few final items left 

- Graphql docs need to be copied - 
image

- algolia search is not working for some reason
- And some of the sections from the documentation need to be copied. e.g. community resources , architecture, comparison, etc.

image

**Comment by KanishkKhurana** [comment]
> Hey @KanishkKhurana 👋
> 
> Great work mate. There are just few final items left
> 
> * Graphql docs need to be copied -
> 
> image
> * algolia search is not working for some reason
> * And some of the sections from the documentation need to be copied. e.g. community resources , architecture, comparison, etc.
> 
> image

-for algolia search i had only changed the process.env variables. the actual values for those variables need to be put in vercel at the time of deployment.


i'll pick the other sections and update them soon.

**Comment by KanishkKhurana** [comment]
@sansyrox All the remaining pages are now added!

**Comment by sansyrox** [comment]
Hey @KanishkKhurana 

image


The community section, hosting, future roadmap and advanced features should be a section like `api reference` 😅

**Comment by KanishkKhurana** [comment]
> Hey @KanishkKhurana
> 
> image
> The community section, hosting, future roadmap and advanced features should be a section like `api reference` 😅

understood, and whatever topics are under them right now should be as it is ? or should they be separate pages ?

**Comment by sansyrox** [comment]
> > Hey @KanishkKhurana
> 
> > 
> 
> > image
> 
> > The community section, hosting, future roadmap and advanced features should be a section like `api reference` 😅
> 
> 
> 
> understood, and whatever topics are under them right now should be as it is ? or should they be separate pages ?

The topics already there should be as they were 😊

**Comment by KanishkKhurana** [comment]
> > > Hey @KanishkKhurana
> > 
> > 
> > > 
> > 
> > 
> > > image
> > 
> > 
> > > The community section, hosting, future roadmap and advanced features should be a section like `api reference` 😅
> > 
> > 
> > understood, and whatever topics are under them right now should be as it is ? or should they be separate pages ?
> 
> The topics already there should be as they were 😊

Understood, On it!

**Comment by KanishkKhurana** [comment]
Screenshot 2023-08-20 at 9 05 52 PM


Hey @sansyrox , all the necessary edits are done! please have a look

**Comment by sansyrox** [comment]
Hey @KanishkKhurana 👋 Great work.

One last request to move the `Client IP Address` section to the API reference section and deleting the `advanced_feature` section.

All looks good otherwise 😄

**Comment by KanishkKhurana** [comment]
Hey @sansyrox ! 
All done, please let me know what you think.

**Review by sansyrox — APPROVED** [review]
LGTM 😄 🔥 

Great work

**Pull request merged**